libr/crypto/p/crypto_bech32.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2017, 2021 Pieter Wuille
```

Comment:
header files dont need a license wall. only code needs that. because those are just interfaces and interfaces are not tied to licensing restrictions